src/solvers/CMakeLists.txt

```diff
@@ -2,26 +2,23 @@
 # other Serac Project Developers. See the top-level LICENSE file for
 # details.
 #
-# SPDX-License-Identifier: (BSD-3-Clause) 
+# SPDX-License-Identifier: (BSD-3-Clause)
+
+add_subdirectory(physics_utilities)
+add_subdirectory(physics_operators)
 
 set(solvers_sources
     base_solver.cpp
-    equation_solver.cpp
     nonlinear_solid_solver.cpp
-    nonlinear_solid_operators.cpp
```

Comment:
thank you.. you have no idea how many times I've clicked on operators instead of solvers and wondered why my line numbers don't match.  I kept meaning to suggest this.